plugins/inputs/disk/README.md

```diff
@@ -15,7 +15,7 @@ Note that `used_percent` is calculated by doing `used / (used + free)`, _not_
   # mount_points = ["/"]
 
   ## Ignore mount points by filesystem type.
-  ignore_fs = ["tmpfs", "devtmpfs", "devfs", "iso9660", "overlay", "aufs", "squashfs"]
+  ignore_fs = ["tmpfs", "devtmpfs", "devfs", "iso9660", "overlay", "aufs", "squashfs", "tracefs"]
```

Comment:
This list out to have even more entries, include at least "fuse.portal" to get rid of stupid "/run/user/1000/doc" warnings.

On my pc I have the following in addition

  ignore_fs = ["tmpfs", "devtmpfs", "devfs", "iso9660", "overlay", "aufs", "squashfs", "cgroup", "sysfs", "proc", "securityfs", "devpts", "pstore", "configfs", "hugetlbfs", "debugfs", "selinuxfs", "mqueue", "rpc_pipefs", "nsfs", "tracefs", "fuse.portal" ]

and I assume all of the additions merits being included upstream.